app/helpers/capturable_helper.rb

```diff
@@ -9,4 +9,25 @@ def display_janrain_widget?
   def devise_resource_name
     Devise.mappings.keys[0].to_s
   end
+
+  def janrain_css_url
+    if Devise.janrain_css_url
+      Devise.janrain_css_url.split(' ').to_json
+    else
+      ['//cdn.oreillystatic.com/members/css/janrain.min.css'].to_json
+    end
+  end
+
+  def janrain_mobile_css_url
+    if Devise.janrain_mobile_css_url
+      Devise.janrain_mobile_css_url.split(' ').to_json
+    else
+      ['//cdn.oreillystatic.com/members/css/janrain.min.css'].to_json
```

Comment:
I noticed that, too, and need to check if it's always been set as this in Capturable. If it has, then I think the only production app it would affect right now is the customer support portal.